### DISCUSSION TITLE
Ender 3 S1 Firmware

### DISCUSSION THREAD

**Discussion by mosheas** [Jyers/Marlin#1703] [Q&A]
It's possible run the firmware with certain settings on  Ender-3 S1?

**Comment by BlueWings172** [comment]
I wonder if this board is better than Creality 4.2.7 board and if this newer board can be used in the Ender 3 V2. is there any advantage of this new board ? I have the 4.2.2 board.

**Reply by RoisinDubh79** [reply]
Bryan of the BV3D YouTube channel did a review recently and he noted that it was a 4.2.3 mainboard.

No additional info with the details you're looking for, but there are some visuals of the actual board.

**Reply by mosheas** [reply]
I believe it's only a matter of time before a source code comes out.

**Reply by mosheas** [reply]
S1 printer is much quality than V2 so it is challenging developing extended firmware

**Reply by RoisinDubh79** [reply]
> 

Oh, maybe that was Igor from My Tech Fun where I saw a 4.2.3 board.

**Comment by paul-aviles** [comment]
I am getting my S1 today and emailed Creality about the sources. They advertise a complete open source printer so will see :-)

**Comment by BlueWings172** [comment]
is the Ender 3 V2 an open source printer ? Did Creality publish 3D model of the V2? There was model that was designed by someone (not Creality) on grabcad and then removed.

Makes me wonder what an open source printer is and if the v2 is one.

**Reply by BlueWings172** [reply]
I have some version of the CAD that was published though it has inaccuracies unfortunately (position of the screw holes in the gantry).

Hope future new products come with official details because they are critical for modding and designing accessories.

**Comment by paul-aviles** [comment]
I know this is not really related to the firmware issue, but wanted to share the S1 I got just in case it helps anyone. The unit looks a lot sturdier that the 3 V2 for sure, but some of the issues I think still persist in my opinion. The pulley system that moves the nozzle is still not completely horizonal in respect to the band, I know it is a small angle, but with the small tolerances these machines need even that small angle pulls the unit upward when close to either left or right side. I wonder how come they have not use a screw already and eliminate the bands. 
![X pulley](https://user-images.githubusercontent.com/8346216/147710509-b9e605ad-6656-4ec9-863a-d007374627a0.jpg)

The other issue I found is with the nozzle/extruder connector. As you can see in the pics the connector shows the pins which can cause a short, so was an easy fix to align it properly. 

![Nozzle Misaligned](https://user-images.githubusercontent.com/8346216/147710578-2b40bcad-66e8-4b48-8cd8-b06c58afc6cc.jpg)
![Nozzle Aligned](https://user-images.githubusercontent.com/8346216/147710582-f4f97122-f9fb-4a74-90c2-258a3cd8b25e.jpg)

Any details which could help with regards of the firmware with the unit let me know. The USB includes Cura 4.9 and regular stuff, nothing marvelous and firmware is version 1.0.3_C... :-(

Paul

**Comment by mosheas** [comment]
I found the engines X, Y to heat up more than usual (Temp 52-57 C) , so I checked Ref voltage and found it was too high. The voltage should be X, Y, 1.1V and Z 1.2V, E 0.95V. now the temp is 42-45 C

**Reply by paul-aviles** [reply]
mosheas, is there a process to review the voltage? The z stepper motor turns very high compared to the ender 3 v2.
